app/master/build.py

```diff
@@ -228,6 +229,27 @@ def complete_subjob(self, subjob_id, payload=None):
             self.mark_failed('Error occurred while completing subjob {}.'.format(subjob_id))
             raise
 
+    def _get_atom_artifact_sys_path(self, subjob_id, atom_id):
+        return os.path.join(
+            self._build_results_dir(),
+            Subjob.ATOM_DIR_FORMAT.format(
+                subjob_id,
+                atom_id,
```

Comment:
I wonder if this code should be more central -- this same `.format()` call is already happening in `Subjob`. Generally it gives me an uncomfortable feeling to have unformatted strings very far away from the place they are formatted -- it's a form of tight coupling.